src/ResourceManager/Storage/Commands.Management.Storage.Test/ScenarioTests/Common.ps1

```diff
@@ -162,7 +158,7 @@ Gets the default location for a provider
 #>
 function Get-ProviderLocation($provider)
 {
-    "westus"
```

Comment:
Instead of a hardcoded location here, please use Get-Location: https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Common.ps1#L583

Reply:
Fixed by change to: 
>Get-Location "Microsoft.Storage" "storageAccounts" "East Asia"